_distro_map.yml

```diff
@@ -32,12 +29,21 @@ openshift-enterprise:
     enterprise-3.0:
       name: '3.0'
       dir: enterprise/3.0
+      distro-overrides:
+        name: OpenShift Enterprise
     enterprise-3.1:
       name: '3.1'
       dir: enterprise/3.1
+      distro-overrides:
+        name: OpenShift Enterprise
     enterprise-3.2:
       name: '3.2'
       dir: enterprise/3.2
+      distro-overrides:
+        name: OpenShift Enterprise
+    container-platform-3.3:
```

Comment:
@vikram-redhat We had said we were gonna stick with `enterprise-3.3` for the actual branch name. Still the case? If so, we'll need to adjust this line.